packages/invoices/src/InboundShipment/DetailView/api.ts

```diff
@@ -382,33 +360,27 @@ export const useInboundLines = (itemId?: string): InvoiceLine[] => {
 };
 
 export const useInboundItems = () => {
-  const { sortBy, onChangeSortBy } = useSortBy<OutboundShipmentSummaryItem>({
+  const { sortBy, onChangeSortBy } = useSortBy<InboundShipmentItem>({
     key: 'itemName',
   });
-  const onSort = (column: Column<OutboundShipmentSummaryItem>) => {
+  const onSort = (column: Column<InboundShipmentItem>) => {
     onChangeSortBy({
       key: column.key,
       isDesc: sortBy.key === column.key ? !sortBy.isDesc : false,
     });
   };
-  const selectItems = useCallback(
-    (invoice: Invoice) => {
-      return inboundLinesToSummaryItems(invoice.lines).sort(
-        getDataSorter(
-          sortBy.key as keyof OutboundShipmentSummaryItem,
-          !!sortBy.isDesc
-        )
-      );
-    },
-    [sortBy]
-  );
+  const selectItems = useCallback((invoice: Invoice) => {
+    return inboundLinesToSummaryItems(invoice.lines).sort(
+      getDataSorter(sortBy.key as keyof InboundShipmentItem, !!sortBy.isDesc)
```

Comment:
this sorting isn't actually working right now. Will need to make a special sort fn where the sort value comes from the lines